### PR TITLE
Rails 4 support

### PR DESCRIPTION
**Pull request by spectator**
Hello again :)

Last pull request in series of Rails 4 update.

Everything is pretty much the same as for responders and has_scope.

Commets are always greatly appreciated.

Thanks.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,9 @@
-source :rubygems
+source 'https://rubygems.org'
 
 gemspec
 
-gem 'rails', '~> 3.1'
-gem 'mocha'
-gem 'turn'
+gem 'rails', '~> 4.0.0.beta1'
```
Comment (joelmoss) [inline Gemfile:5]: We don't want to make Rails 4 beta a requirement. Maybe when final is released we can do that. But a beta release should not be the required version.

**Comment by joelmoss** [comment]
We want to support Rails4, but we should not be requiring it until the final is released.

**Comment by spectator** [comment]
Oh well, you can reopen it when Rails 4 is released.

**Comment by whatthewhat** [comment]
@joelmoss Hi, would `gem 'rails', '>= 3.1'` be too open?
I'm sure a lot of people would like to use inherited_resources with rails 4 beta

**Comment by joevandyk** [comment]
Time to reopen this?

**Comment by spectator** [comment]
@joevandyk 

I guess you can use master branch if you need Rails 4 support and it will work just fine since it has its dependencies updated to release candidates of responders and has_scope.

My branch (https://github.com/spectator/inherited_resources/commits/rails4) was mostly dealing with dependencies (fixed in master) and tests (probably not fixed yet).

**Comment by preston** [comment]
Has support been added to the mainline? If not, please reopen! :+1:

**Comment by maioral** [comment]
+1

**Comment by shlensky** [comment]
+1

**Comment by versilov** [comment]
+1

**Comment by gra** [comment]
+1

**Comment by adamrneary** [comment]
+1, whether this PR or another, it would be good to know when we have stable rails 4 support for this lib.

**Comment by fny** [comment]
:+1:

**Comment by preston** [comment]
Anyone happen to have a patch branch yet? I bet this is a big issue for a lot people.

**Comment by spectator** [comment]
Have you tried using master branch? I think it should work fine.

``` ruby
gem 'inherited_resources', github: 'josevalim/inherited_resources'
```

**Comment by preston** [comment]
Ok, I think I've got it working on the master head code, though it took me a few to figure out how to define the permitted_params method correctly. Apparently I didn't RTFM thoroughly. :)  Thanks!

**Pull request closed without merging**